pkg/Microsoft.Windows.Compatibility/Microsoft.Windows.Compatibility.pkgproj

```diff
@@ -50,18 +51,26 @@
     <PrereleaseLibraryPackage Include="System.Text.Encoding.CodePages" />
     <PrereleaseLibraryPackage Include="System.Threading.AccessControl" />
 
-    <!-- External pre-release packages -->
+    <!-- Service model packages -->
     <LibraryPackage Include="System.ServiceModel.Primitives">
-      <Version>4.4.1-servicing-25917-01</Version>
+      <Version>$(ServiceModelVersion)</Version>
+    </LibraryPackage>
+    <LibraryPackage Include="System.ServiceModel.Duplex">
```

Comment:
Can you please verify that these new packages don't lift any of the corefx packages?